### ISSUE TITLE
Updates check message remains in notification area although nothing to update

### ISSUE THREAD

**Issue by pled** [ImranR98/Obtainium#1066]
I have noticed that since a few weeks (Android 14 ?), notification when Obtainium is checking for updates keeps displaying in the notification area, although no app needs any update : 

![Screenshot_20231103-195431](https://github.com/ImranR98/Obtainium/assets/2879483/73484ddd-a6bc-4bee-963f-ef67e2d916ea)

The Obtainium icon has also a notification badge at that time, which makes you believe an update is required for some app.
But there is no application to update, so I was expecting that notification disappear once operation is complete. 
I have to swipe notification to the left to make it disappear.

Am I missing something ?

**Please complete the following information:**
 - Device: Pixel 6a (android 14)
 - OS: GrapheneOS 
 - Obtainium Version: 0.14.31

**Comment by ImranR98** [comment]
Duplicate of #608.
